### PR TITLE
Add Monkey Island music

### PR DESCRIPTION
**Pull request by SlyryD**
Samples: https://www.youtube.com/playlist?list=PLJxeyhslPQZ1SpActG-FI0NazqO4fCO-t

### CODE REVIEW

File: data/Music/Monkey Island Series/The Secret Of Monkey Island/Theme.meta [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The Secret of Monkey Island - Theme
+0x03
```
Comment (SlyryD) [inline data/Music/Monkey Island Series/The Secret Of Monkey Island/Theme.meta:2]: I don't actually know what this number means. Can someone help me out with that?

Comment (SlyryD) [reply data/Music/Monkey Island Series/The Secret Of Monkey Island/Theme.meta:2]: Is this maybe to distinguish fanfares from background music?

Comment (darkangel93x) [reply data/Music/Monkey Island Series/The Secret Of Monkey Island/Theme.meta:2]: The number distinguishes which instrumentalist the game is supposed to use.

Comment (SlyryD) [reply data/Music/Monkey Island Series/The Secret Of Monkey Island/Theme.meta:2]: Ah okay, so this is main orchestra. How does this value interact with the instruments in the midi file?

Comment (darkangel93x) [reply data/Music/Monkey Island Series/The Secret Of Monkey Island/Theme.meta:2]: it says the rando what instrument list to load.

and depending on the list chosen the values in the sequence file have to be changed cause there are different instruments obviously

**Comment by darkangel93x** [comment]
could you provide  me the original midi to this? would love to work something out with it :)

File: data/Music/Monkey Island Series/The Secret Of Monkey Island/Theme.meta (hunk 1 above)
Comment (SlyryD) [reply data/Music/Monkey Island Series/The Secret Of Monkey Island/Theme.meta:2]: Thanks for the help!
-- I switched up the instruments for the Secret of Monkey Island theme: https://www.youtube.com/watch?v=JH6VoP8QM7A
-- I added a Fanfare from Monkey Island 2 (Chapter Screen): https://www.youtube.com/watch?v=vDaztQ67MQk
-- I also switched up the Monkey Island 2 Theme

Unfortunately, the Monkey Island 2 Theme is now too large to be played in most places, so I plan to trim that down at some point in the future.

**Pull request merged**